documentation/quarkus/getting-started-quarkus.asciidoc

```diff
@@ -0,0 +1,235 @@
+= Quarkus Quickstart 
+
+This guide will show a quickstart how to create a Quarkus app and will briefly show the key functionalities that Quarkus provides and how to start as a beginner or also experienced developers.
+
+
+== Introduction to Quarkus 
+
+To get the first introduction to Quarkus you can read the link:../quarkus.asciidoc[Quarkus introduction guide]. To get a short overview where you can find the important Quarkus related guides follow the chapter **Guide to the Reader**.
+Also, a comparison of the link:../quarkus.asciidoc#Pros[advantages] and link:../quarkus.asciidoc#cons[disadvantages] of a Quarkus application compared to the alternative framework Spring.
+This comparison will be supported by our link:../performance-comparison-spring-quarkus.asciidoc[performance comparison between Spring and Quarkus] that shows the lower resource consumption and startup time of Quarkus applications.
+
+
+== Installation of Tools and Dependencies
+
+First, we need to install some dependencies and tools before we can start programming. Our tool https://devonfw.com/website/pages/docs/devonfw-ide-introduction.asciidoc.html[devonfw-ide] comes with many development tools, frameworks, and libraries for you.
+We will install the devonfw-ide with the following tools, that are needed for this guide.
+
+* Maven 
+* Java
+* any IDE (devonfw-ide supports Eclipse, Intellij and VScode)
+* CobiGen
+* Docker 
+
+We recommend installing the devonfw-ide but if you already got your system configured and the tools from above installed you can skip to <<Bootstrap a Quarkus Project>>.
```

Comment:
Isn't this already a working link ?

Reply:
This syntax isn't allowed. It will break during processing to PDF and website: https://github.com/devonfw/docgen/blob/master/documentation/guidelines.asciidoc